packages/connector/implementation/src/profiles/BigQueryDbtProfile.kt

```diff
@@ -0,0 +1,60 @@
+package io.tellery.connectors.profiles
+
+import annotations.Dbt
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.google.gson.Gson
+import io.tellery.annotations.Config
+import io.tellery.connectors.BigQueryConnector
+import io.tellery.entities.Profile
+import java.util.*
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Dbt(
+    type = "BigQuery",
+    configs = [
+        Config(
+            name = "Git Url",
+            type = Config.ConfigType.STRING,
+            description = "The git url of your dbt repository. The format should be: git://[endpoint]:[organization_name]/[repo_name].git.",
+            hint = "",
+            required = false
+        ),
+        Config(
+            name = "Dbt Project Name",
+            type = Config.ConfigType.STRING,
+            description = "The name of your dbt project, it should be equal with the name in dbt_project.yml.",
+            hint = "",
+            required = false
+        ),
+        Config(
+            name = "Public Key",
+            type = Config.ConfigType.STRING,
+            description = "The public key generated by tellery for dbt repository, please copy it to the 'deploy key' in git server.",
+            hint = "",
+            required = false
+        )
```

Comment:
Since all configuration entries of Dbt annotation are the same, maybe they can be moved to the definition of Dbt annotation?

Reply:
Sure, the next PR will put all config or the key name to a constant class.